CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v1.1.1 - Abu (Patch) [xxxx-xx-xx]
```

Comment:
Just a gentle reminder to add the date. (Perhaps you were planing to add that after approval of PR?)

Reply:
Yupe, I usually do that after the approval :)